console/src/view/resources.rs

```diff
@@ -0,0 +1,185 @@
+                    dur_cell(resource.total(now)),
+                    Cell::from(target_width.update_str(resource.target()).to_owned()),
+                    Cell::from(type_width.update_str(resource.concrete_type()).to_owned()),
+                    Cell::from(location_width.update_str(resource.location())),
+                    Cell::from(Spans::from(
+                        resource
+                            .formatted_attributes()
+                            .iter()
+                            .flatten()
+                            .cloned()
+                            .collect::<Vec<_>>(),
+                    )),
+                ]);
+
+                if resource.dropped() {
+                    row = row.style(styles.terminated());
+                }
+
+                Some(row)
+            })
+        };
+
+        let (selected_style, header_style) = if let Some(cyan) = styles.color(Color::Cyan) {
+            (Style::default().fg(cyan), Style::default())
+        } else {
+            (
+                Style::default().remove_modifier(style::Modifier::REVERSED),
+                Style::default().add_modifier(style::Modifier::REVERSED),
+            )
+        };
+        let header_style = header_style.add_modifier(style::Modifier::BOLD);
+
+        let header = Row::new(header.iter().enumerate().map(|(idx, &value)| {
+            let cell = Cell::from(value);
+            if idx == self.selected_column {
+                cell.style(selected_style)
+            } else {
+                cell
+            }
+        }))
+        .height(1)
+        .style(header_style);
+
+        let table = if self.sort_descending {
+            Table::new(rows)
+        } else {
+            Table::new(rows.rev())
+        };
+
+        let block = styles
+            .border_block()
+            .title(vec![bold(format!("Resources ({}) ", self.len()))]);
+
+        let layout = layout::Layout::default()
+            .direction(layout::Direction::Vertical)
+            .margin(0);
+
+        let chunks = layout
+            .constraints(
+                [
+                    layout::Constraint::Length(1),
+                    layout::Constraint::Min(area.height - 1),
+                ]
+                .as_ref(),
+            )
+            .split(area);
+        let controls_area = chunks[0];
+        let tasks_area = chunks[1];
+
+        let attributes_width = layout::Constraint::Percentage(100);
+        let widths = &[
+            id_width.constraint(),
+            kind_width.constraint(),
+            layout::Constraint::Length(DUR_LEN as u16),
+            target_width.constraint(),
+            type_width.constraint(),
+            location_width.constraint(),
+            attributes_width,
+        ];
+
+        let table = table
+            .header(header)
+            .block(block)
+            .widths(widths)
+            .highlight_symbol(">> ")
```

Comment:
nit/TIOLI: maybe worth making this a const in the `view` module, since we use the same symbol here as in the tasks list? that way, both modules could just say
```suggestion
            .highlight_symbol(view::TABLE_HIGHLIGHT_SYMBOL)
```
or something? that way, if we change it, we would only have to change it in one place.

alternatively, this could be a method on `Styles`, which might be nice if we wanted to use a unicode character instead of `>> ` when UTF-8 is enabled...but that should be a separate change.